cyclops-ctrl/internal/cluster/k8sclient/client.go

```diff
@@ -559,6 +561,28 @@ func (k *KubernetesClient) mapConfigMap(group, version, kind, name, namespace st
 	}, nil
 }
 
+func (k *KubernetesClient) mapPersistentVolumeClaims(group, version, kind, name, namespace string) (*dto.PersistentVolumeClaim, error) {
+	persistentvolumeclaim, err := k.clientset.CoreV1().PersistentVolumeClaims(namespace).Get(context.Background(), name, metav1.GetOptions{})
+	if err != nil {
+		return nil, err
+	}
+
+	storage = 0
+	if persistentvolumeclaim.Spec.Resources != nil && persistentvolumeclaim.Spec.Resources.Requests != nil {
+		storage = persistentvolumeclaim.Spec.Resources.Requests.Storage
+	}
```

Comment:
Size is a string, so you can default to an empty string. My bad I said 0 in a previous comment 😬 

Also, you will need to initialize storage with `:=`. Another thing is that you are checking for Resources and Resources. Requests, but you need to check for Requests and Resources.Storage()

Storage is a function instead of a field so you need to call it to retrieve Storage object. Then you need to call String() from storage to get it in a string format

```suggestion
	storage := ""
	if persistentvolumeclaim.Spec.Resources.Requests != nil && persistentvolumeclaim.Spec.Resources.Requests.Storage() != nil {
		storage = persistentvolumeclaim.Spec.Resources.Requests.Storage().String()
	}
```